/*
 * This file was automatically generated by EvoSuite
 * Thu Aug 24 19:10:20 GMT 2023
 */

package com.example.Amit.exception;

import org.junit.Test;
import static org.junit.Assert.*;
import com.example.Amit.exception.ErrorResponse;
import com.example.Amit.exception.SectorControllerExceptionHandler;
import com.example.Amit.exception.SectorNotFoundException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;
import org.springframework.http.ResponseEntity;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class SectorControllerExceptionHandler_ESTest extends SectorControllerExceptionHandler_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      SectorControllerExceptionHandler sectorControllerExceptionHandler0 = new SectorControllerExceptionHandler();
      SectorNotFoundException sectorNotFoundException0 = new SectorNotFoundException();
      ResponseEntity<ErrorResponse> responseEntity0 = sectorControllerExceptionHandler0.handleCompanyNotFoundException(sectorNotFoundException0);
      assertEquals(404, responseEntity0.getStatusCodeValue());
  }
}
